### PR TITLE
CPLP-1134 add notification to db

### PR DESCRIPTION
**Pull request by Phil91**
Also see [PR in portal backend](https://github.com/catenax-ng/product-portal-backend/pull/110)

### CODE REVIEW

File: postgres/init/20-portal.sql [hunk 1]
```diff
@@ -638,6 +638,44 @@ CREATE TABLE portal.languages (
...
+);
+
+
+--
+-- Name: notifications; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.notifications (
+    id uuid NOT NULL,
+    company_user_id uuid NOT NULL,
```
Comment (jjeroch) [inline postgres/init/20-portal.sql:667]: under discussion

File: postgres/init/20-portal.sql [hunk 2]
```diff
@@ -638,6 +638,44 @@ CREATE TABLE portal.languages (
...
+    date_created timestamp with time zone NOT NULL,
+    title character varying(255) NOT NULL,
```
Comment (jjeroch) [inline postgres/init/20-portal.sql:669]: I suggest that we shorten this to actual 150 digits. In many cases pop-up messages are limited to 150-160 characters. Thats in the mobile world the standard as far as I know. Off course we dont have smartphones in focus, but already thinking about being capable to fit into those standards.
I also created a few examples which are currently all running with 60-100 characters.

File: postgres/init/20-portal.sql (hunk 1 above)
Comment (Phil91) [reply postgres/init/20-portal.sql]: changed the naming of the user_ids

File: postgres/init/20-portal.sql (hunk 2 above)
Comment (Phil91) [reply postgres/init/20-portal.sql]: i think thats a good idea. i changed it to 150 digits

Comment (jjeroch) [reply postgres/init/20-portal.sql]: Merci

**Review by evegufy — APPROVED** [review]

**Review by ntruchsess — APPROVED** [review]

**Review by ntruchsess — APPROVED** [review]
